### ISSUE TITLE
Chat window very slow when using @Files or highlighting code

### ISSUE THREAD

**Issue by KungKranium** [continuedev/continue#2928]
### Before submitting your bug report

- [X] I believe this is a bug. I'll try to join the [Continue Discord](https://discord.gg/NWtdYexhMs) for questions
- [X] I'm not able to find an [open issue](https://github.com/continuedev/continue/issues?q=is%3Aopen+is%3Aissue) that reports the same bug
- [X] I've seen the [troubleshooting guide](https://docs.continue.dev/troubleshooting) on the Continue Docs

### Relevant environment info

```Markdown
- OS:macOs
- Continue version:v0.81
- IDE version:Intellij 2024.2.0.1 Community
- Model: Gemini pro
- config.json:
```

### Description

Sometimes, not always, continue is slow, as in lagging when I'm writing in the chat window, or when its generating responses. Clicking outside the continue window, such as in an editor tab, makes it go full speed again. This is shown by either slowly accepting inputs or the circling lights around the question box circling laggish.

### To reproduce

1. Use continue
2. Notice lag

**Comment by sestinj** [comment]
@KungKranium is this newly happening on 0.0.81, and not on 0.0.80? Have a theory about the cause of this, but it would help to confirm that earlier versions are fine. Will hopefully have an update for you by end of week

**Comment by KungKranium** [comment]
> @KungKranium is this newly happening on 0.0.81, and not on 0.0.80? Have a theory about the cause of this, but it would help to confirm that earlier versions are fine. Will hopefully have an update for you by end of week

No, I had this even back on 0.0.78 (or similar, can't remember exact version I had), I updated to .81 in hopes of the issue solving

**Comment by RomneyDa** [comment]
@KungKranium are you still experiencing this on the newest prerelease?
